### PR TITLE
Remove variable expansion from tftp autoyast test

### PR DESCRIPTION
**Pull request by rwx788**
In aytests framework variables expansion is implemented, which doesn't
exist in openQA. But setup is static and we precisely know ip address of
supportserver. All sources are copied one to one from aytests repo.
We can get rid of support server here and upload tftp script simply by
using worker as we do for autoinst profiles. As of now, we make it this
way.

See aytests repo: https://github.com/yast/aytests-tests/blob/master/aytests/tftp.vars

Requires AUTOYAST to have value autoyast_sle12sp2/tftp.xml for autoyast_aytests_tftp+supportserver test suite.

Verification run: http://gershwin.arch.suse.de/tests/517

### CODE REVIEW

File: data/autoyast_sle12sp2/post_script.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
```
Comment (okurz) [inline data/autoyast_sle12sp2/post_script.sh]: how about `#!/bin/sh -ex` ?

File: data/autoyast_sle12sp2/post_script.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+set -x
+echo "This is an AutoYaST post script test logfile" >/var/log/autoyast_post_script.log || exit 1
+exit 0
```
Comment (okurz) [inline data/autoyast_sle12sp2/post_script.sh]: This line should not be necessary

Comment (rwx788) [inline data/autoyast_sle12sp2/post_script.sh]: Good point, fixed.

File: data/autoyast_sle12sp2/post_script.sh (hunk 1 above)
Comment (rwx788) [inline data/autoyast_sle12sp2/post_script.sh]: Sounds good. I got this script from aytests repo, so we should back merge it. I'll mention this in the progress ticket https://progress.opensuse.org/issues/20064

**Pull request merged**